.github/workflows/coverage.yml

```diff
@@ -101,4 +102,4 @@ jobs:
               repo: context.repo.repo,
               issue_number: context.issue.number,
               body: commentBody,
-            });
+            });
```

Comment:
please revert.

Reply:
added this as sarah suggested [here](https://github.com/0saurabh0/django/pull/6#issuecomment-3219300658)
Should I remove?

Reply:
ah, my bad.
will do.